packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.scss

```diff
@@ -18,20 +18,21 @@
 }
 
 ::slotted(input) {
-  flex: 1 1 auto;
+  flex: 0 1 auto;
   align-self: auto;
   order: 0;
-  width: var(--gse-ui-formControl-input-colorPicker-size);
-  height: var(--gse-ui-formControl-input-colorPicker-size);
+  min-width: var(--gse-ui-colorPicker-input-swatchSizing);
```

Comment:
I added this min-width so that the aspect ratio can be 24x24 for the swatch. Without the min-width the swatch aspect ratio is 22.2x24 (I believe this is because the browser automatically sets a default width to the input, so this min-width style is a way to override that)